FRAuth/FRAuthTests/FRAuthSwiftTests/E2ETests/Callback-Live/AA_07_AppIntegrityTest.swift

```diff
@@ -177,7 +921,7 @@ final class AA_07_AppIntegrityTest: CallbackBaseTest {
         // Provide input value for the username collector callback
         for callback in node.callbacks {
             if callback is NameCallback, let nameCallback = callback as? NameCallback {
-                nameCallback.setValue(AA_07_AppIntegrityTest.USERNAME)
+                nameCallback.setValue(AA_05_DeviceBindingCallbackTest.USERNAME)
```

Comment:
should this be `AA_07_AppIntegrityTest.USERNAME`?

Reply:
Fixed.